Add sagaevm

VIJ-VincentJacquot opened the pull request:
adding sagaEVM as a supported network on Keplr

editaahn commented on evm/eip155:5464.json:
```diff
@@ -0,0 +1,102 @@
+        "coinDenom": "GAS",
+        "coinMinimalDenom": "gas",
+        "coinDecimals": 18
+      }
```
@VIJ-VincentJacquot  sorry for the inconvenience, but to avoid potential management issues, I would greatly appreciate it if you could follow the rules.

editaahn commented:
Did you rebase your branch with the latest changes? It seems something went wrong

VIJ-VincentJacquot commented:
hey @editaahn - yes rebased the branch and sounds like some paths were broken somehow. 

I fixed some paths and added a new token as well. If it does not work again for some reason I'll close this PR and start from fresh. Thanks!

editaahn commented:
Hey @VIJ-VincentJacquot  would you mind submitting a new PR on the latest fast-forward code base or just leaving your own changes here?
